lib/perl/Genome/Model/CwlPipeline/Command/Restart.pm

```diff
@@ -32,6 +32,16 @@ sub execute {
     my $build_count = scalar(@builds);
     my @errors;
     for my $build (@builds) {
+        my $anp = $build->model->analysis_project;
```

Comment:
Should we put this inside `_restart_build` so that each build can be independently tried?